modules/1plusXRtdProvider.js

```diff
@@ -14,7 +14,7 @@ const ORTB2_NAME = '1plusX.com'
 const PAPI_VERSION = 'v1.0';
 const LOG_PREFIX = '[1plusX RTD Module]: ';
 const OPE_FPID = 'ope_fpid'
-const LEGACY_SITE_KEYWORDS_BIDDERS = ['appnexus'];
+const LEGACY_SITE_KEYWORDS_BIDDERS = ['dummy'];
```

Comment:
```suggestion
```

rather than changing the value to dummy here we can completely remove this variable and all associated code in lines 155 to 164